scripts/prepare_test_weights.py

```diff
@@ -0,0 +1,577 @@
+"""
+Download and convert weights for testing
+
+To see what weights will be downloaded and converted, run:
+DRY_RUN=1 python scripts/prepare_test_weights.py
+"""
+import hashlib
+import os
+import subprocess
+import sys
+from urllib.parse import urlparse
+
+import requests
+from tqdm import tqdm
```

Comment:
Since you import those directly you should add them explicitly as dependencies in the `conversion` extra.

Reply:
done